ush/parsing_namelists_CICE.sh

```diff
@@ -435,51 +256,51 @@ cat > ice_in <<eof
    f_Tref          = 'mdhxx' 
    f_Qref          = 'mdhxx'
    f_congel        = 'mdhxx' 
-   f_frazil        = 'x' 
-   f_snoice        = 'x' 
-   f_dsnow         = 'x' 
-   f_melts         = 'x'
+   f_frazil        = 'mdhxx' 
+   f_snoice        = 'mdhxx' 
+   f_dsnow         = 'mdhxx' 
+   f_melts         = 'mdhxx'
    f_meltt         = 'mdhxx'
    f_meltb         = 'mdhxx'
    f_meltl         = 'mdhxx'
-   f_fresh         = 'x'
+   f_fresh         = 'mdhxx'
    f_fresh_ai      = 'x'
-   f_fsalt         = 'x'
+   f_fsalt         = 'mdhxx'
    f_fsalt_ai      = 'x'
-   f_fbot          = 'x'
-   f_fhocn         = 'x' 
+   f_fbot          = 'mdhxx'
+   f_fhocn         = 'mdhxx' 
    f_fhocn_ai      = 'x' 
    f_fswthru       = 'x' 
    f_fswthru_ai    = 'x' 
    f_fsurf_ai      = 'x'
    f_fcondtop_ai   = 'x'
    f_fmeltt_ai     = 'x' 
-   f_strairx       = 'mdhxx' 
-   f_strairy       = 'mdhxx' 
+   f_strairx       = 'mdhxx'
+   f_strairy       = 'mdhxx'
    f_strtltx       = 'x' 
    f_strtlty       = 'x' 
    f_strcorx       = 'x' 
    f_strcory       = 'x' 
-   f_strocnx       = 'x' 
-   f_strocny       = 'x' 
+   f_strocnx       = 'mdhxx' 
+   f_strocny       = 'mdhxx' 
    f_strintx       = 'x' 
    f_strinty       = 'x'
    f_taubx         = 'x'
    f_tauby         = 'x'
    f_strength      = 'x'
-   f_divu          = 'mdhxx'
-   f_shear         = 'mdhxx'
+   f_divu          = 'x'
```

Comment:
let's revert any that have gone from "on" to "off" (mdhxx->x), but anything that was turned on (x->mdhxx) as on.  

So the variables to revert are: f_divu, f_shear, f_divdt* (lines 296-299), f_mld_onset, f_frz_onset

Reply:
Done